### PR TITLE
Add IT case: customized-tuners-demotuner

### PR DESCRIPTION
**Pull request by J-shang**
Add IT case to test user installed builtin tuner.

### CODE REVIEW

File: test/config/integration_tests.yml [hunk 1]
```diff
@@ -242,3 +242,9 @@ testCases:
 
 - name: tuner-metis
   configFile: test/config/tuners/metis.yml
+
+#########################################################################
+# nni customized-tuners test
+#########################################################################
+- name: customized-tuners-demotuner
+  configFile: test/config/customized_tuners/demotuner-sklearn-classification.yml
```
Comment (chicm-ms) [inline test/config/integration_tests.yml:250]: missing end of line.

File: test/pipelines/pipelines-it-frameworkcontroller.yml [hunk 2]
```diff
@@ -32,6 +32,12 @@ jobs:
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=SMAC
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=BOHB
     displayName: 'Install dependencies for integration tests in frameworkcontroller mode'
+  
+  - script: |
+      cd examples/tuners/customized_tuner
+      python3 setup.py develop
+      nnictl package install ./
```
Comment (chicm-ms) [inline test/pipelines/pipelines-it-frameworkcontroller.yml:39]: please set PATH like `PATH=$HOME/.local/bin:$PATH nnictl package install --name=SMAC`
and add `set -e` at beginning.

File: test/pipelines/pipelines-it-paiYarn.yml [hunk 3]
```diff
@@ -29,6 +29,12 @@ jobs:
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=BOHB
     displayName: 'Install dependencies for integration tests in PAI mode'
 
+  - script: |
+      cd examples/tuners/customized_tuner
```
Comment (chicm-ms) [inline test/pipelines/pipelines-it-paiYarn.yml:33]: paiYarn is obsolete,  please remove this change.

File: test/pipelines/pipelines-it-frameworkcontroller.yml (hunk 2 above)
Comment (J-shang) [reply test/pipelines/pipelines-it-frameworkcontroller.yml:39]: Thank you, I will modify them.

File: test/pipelines/pipelines-it-local.yml [hunk 4]
```diff
@@ -21,6 +21,12 @@ jobs:
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=BOHB
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=PPOTuner
     displayName: 'Install dependencies for integration tests'
+  - script: |
+      set -e
+      cd examples/tuners/customized_tuner
+      python3 setup.py develop
```
Comment (chicm-ms) [inline test/pipelines/pipelines-it-local.yml:27]: Please check this error log:  https://msrasrg.visualstudio.com/NNIOpenSource/_build/results?buildId=14256&view=logs&j=75aeeae9-dab9-55a1-b144-0ab48bfd0d00&t=3f120a8d-7233-5989-6cfa-cd23207edde0

It seems you need to add `--user` here

**Review by chicm-ms — APPROVED** [review]

**Review by SparkSnail — APPROVED** [review]

**Pull request merged**